### PR TITLE
DOC/MAINT: signal: fix some typos

### PR DESCRIPTION
**Pull request by musvaage**
@lucascolley

dumped directory strings exposed a few typos

scipy/signal/windows/_windows.py

fixed typo in the following and changed to lowercase

```diff
-     The Kaiser window is a very good approximation to the Digital Prolate
-     Spheroidal Sequence, or Slepian window, which is the transform which
+     The Kaiser window is a very good approximation to the discrete prolate
+     spheroidal sequence, or Slepian window, which is the transform which
```

Is this comment correct, i.e collection not a sequence but sequences?

```
$ sed -n '2035,2044p' scipy/scipy/signal/windows/_windows.py
    # Here we want to set up an optimization problem to find a sequence
    # whose energy is maximally concentrated within band [-W,W].
    # Thus, the measure lambda(T,W) is the ratio between the energy within
    # that band, and the total energy. This leads to the eigen-system
    # (A - (l1)I)v = 0, where the eigenvector corresponding to the largest
    # eigenvalue is the sequence with maximally concentrated energy. The
    # collection of eigenvectors of this system are called Slepian
    # sequences, or discrete prolate spheroidal sequences (DPSS). Only the
    # first K, K = 2NW/dt orders of DPSS will exhibit good spectral
    # concentration
$
```

### CODE REVIEW

File: scipy/signal/windows/_windows.py [hunk 1]
```diff
@@ -1151,8 +1151,8 @@ def kaiser(M, beta, sym=True):
 
     The Kaiser was named for Jim Kaiser, who discovered a simple approximation
     to the DPSS window based on Bessel functions.
-    The Kaiser window is a very good approximation to the Digital Prolate
-    Spheroidal Sequence, or Slepian window, which is the transform which
+    The Kaiser window is a very good approximation to the discrete prolate
+    spheroidal sequence, or Slepian window, which is the transform which
```
Comment (tylerjereddy) [inline scipy/signal/windows/_windows.py:1155]: this one required me to check Wikipedia, but seems ok: https://en.wikipedia.org/wiki/Window_function

the rest of the changes seem straightforward typo fixes

**Review by tylerjereddy — APPROVED** [review]
thanks

File: scipy/signal/windows/_windows.py (hunk 1 above)
Comment (musvaage) [reply scipy/signal/windows/_windows.py:1155]: _cf_: aberrant acronym expansion

an emeritus professor at Stanford University

**Comment by musvaage** [comment]
@tupui 

integrity of science

the `sed` output language should be reviewed

**Comment by musvaage** [comment]
> the `sed` output language should be reviewed 

@martinholters

Would you mind chipping in on this?

The language doesn't look right to me.

**Comment by tupui** [comment]
@musvaage that's not my area sorry 😅

**Comment by rkern** [comment]
The language there is fine. It's talking in pure math terms, not Python objects.

**Comment by martinholters** [comment]
Not sure what's expected of me, here, but I didn't spot anything obvious in the quoted text.

**Comment by musvaage** [comment]
@martinholters 

First of all thank you for responding as I know you're from a different coding community.

Lines 1152 - 1157 from the file include the singular form `sequence`, i.e. `the DPSS window` (a.k.a. …`Slepian window`).

@tylerjereddy made a review of the patch, so please click "Show resolved" so as reveal his review.

Evidently the prior to this patch expanded acronym had `Digital` rather than `Discrete` for `DPSS`.

Line 1851 includes the plural `Sequences` followed by the singular `DPSS.`

> Is this comment correct, i.e collection not a sequence but sequences?

Lines 2035 - 2044 from the file include the following:

`The collection of eigenvectors of this system are called Slepian sequences, or discrete prolate spheroidal sequences (DPSS). Only the first K, K = 2NW/dt orders of DPSS will exhibit good spectral concentration`

If `sequences` and not **sequence** is factually correct then English language convention would not be `DPSS` but **DPSSs**.

Line 2045 provides a reference URL and @tylerjereddy in his review provided an additional reference URL.

<hr>

> not Python objects

Noted from @rkern, but the pure math description might be improved.

**Comment by martinholters** [comment]
I see... First, "discrete" is definitely better than "digital" in this context. I do agree that for the plural, the acronym should probably be pluralized as DPSSs.

Let me go through the referenced sections:
https://github.com/scipy/scipy/blob/f054bfe5514f35dd47f06b0c2f762b7a857a63b0/scipy/signal/windows/_windows.py#L2039-L2045
DPSS -> DPSSs twice
(Tangetially, the term "order" for DPSSs is not introduced here. But from the context, its meaning is somewhat self-explaining, and the given Wikipedia links gives a definition.)
https://github.com/scipy/scipy/blob/f054bfe5514f35dd47f06b0c2f762b7a857a63b0/scipy/signal/windows/_windows.py#L1154-L1155
Strictly speaking, this should probably be the "0th-order DPSS" or "DPSS window". Maybe make this "discrete prolate spheroidal sequence of order 0, or Slepian window"? A bit of a mouthful, though; maybe better leave as is?
https://github.com/scipy/scipy/blob/f054bfe5514f35dd47f06b0c2f762b7a857a63b0/scipy/signal/windows/_windows.py#L1851-L1856
DPSS -> DPSSs twice

Hope this helps.

**Comment by musvaage** [comment]
Maybe at some point of time this will be revisited in a new Issue.

It will suffice to say herewith that A.T Walden uses a specific phrase pertaining to the subject at hand.

DOI: [10.1016/0165-1684(89)90043-1](https://doi.org/10.1016/0165-1684(89)90043-1)

Abstract
…Kaiser produced a Bessel approximation to the continuous prolate spheroidal wave function.

Zusammenfassung
…Von Kaiser wurde eine Bessel-Approximation für kontinuierliche Prolate Spheroidal Wave Funktion vorgeschlagen.

Résumé
…Kaiser a suggeré une approximation de Bessel à la fonction d'onde continue spheroïdale aplatie.

> Hope this helps.

👍

<hr>

Were editing of the file's math description eventually pursued `WT` in the below should be corrected.

```
$ sed -n '1912,1914p' scipy/scipy/signal/windows/_windows.py
    .. [1] Percival DB, Walden WT. Spectral Analysis for Physical Applications:
       Multitaper and Conventional Univariate Techniques.
       Cambridge University Press; 1993.
$ 
```

**Pull request merged**